src/xdocs/config_regexp.xml

```diff
@@ -145,43 +145,43 @@
         <source>
           // This code is copyrighted
```

Comment:
http://checkstyle.sourceforge.net/config_regexp.html#Regexp

look at page, and remove all leading spaces in this line.
Run "mvn site -Pno-validations" to generate site and see how page become different.

Reply:
Done